habit-tracker/C11/asiu3209/project2/src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import "./index.css";
+import HomePage from "./Pages/HomePage";
+import NavBar from "./components/NavBar";
+import StatisticsPage from "./Pages/StatisticsPage";
+import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
+import { Habit, StoredHabit } from "./components/HabitCard";
+function App() {
```

Comment:
Usually we add whitespace (an empty line) between the imports and the rest of the code.